redfish-core/lib/power_supply_metrics.hpp

```diff
@@ -43,8 +43,8 @@ static const char* historyMaximumInterface =
 static const char* historyAverageInterface =
     "org.open_power.Sensor.Aggregation.History.Average";
 
-static std::array<const char*, 2> historyInterfaces{historyMaximumInterface,
-                                                    historyAverageInterface};
+static std::array<std::string_view, 2> historyInterfaces{
+    historyMaximumInterface, historyAverageInterface};
```

Comment:
Since `historyMaximumInterface` & `historyAverageInterface` are used only for constructing `historyInterfaces`, I think we can simply write it as

```
static constexpr std::array<std::string_view, 2> historyInterfaces{
    "org.open_power.Sensor.Aggregation.History.Maximum",
     "org.open_power.Sensor.Aggregation.History.Average" };
```

Reply:
This change is changing to std::string_view, I think outside this commit